ansible/hosts/lxd_hosts.txt

```diff
@@ -1,27 +1,27 @@
 #central servers (ubuntu lxd containers)
 [cs_servers]
-xroad-lxd-cs ansible_connection=lxd
+xroadlxdcs ansible_connection=lxd
 
 #configuration proxies (ubuntu lxd containers, optional)
 [cp_servers]
-#xroad-lxd-cp ansible_connection=lxd
+#xroadlxdcp ansible_connection=lxd
 
 #certification authority, time stamping authority and ocsp service server for testing purposes (ubuntu)
 [ca_servers]
-xroad-lxd-cs ansible_connection=lxd
+xroadlxdcs ansible_connection=lxd
 
 #security servers (ubuntu lxd containers)
 [ss_servers]
-xroad-lxd-ss1 ansible_connection=lxd
+xroadlxdss1 ansible_connection=lxd
 #for a more realistic setup, add at least one additional server
-#xroad-lxd-ss2 ansible_connection=lxd
+#xroadlxdss2 ansible_connection=lxd
 
 [ss_servers:children]
 centos_ss
 
 #security servers (centos lxd containers, not supported in variant ee)
 [centos_ss]
-#xroad-lxd-rh-ss1 ansible_connection=lxd
+#xroadlxdrhss1 ansible_connection=lxd
```

Comment:
I did not encounter any issues with using LXD4 from copr-lxc4 or LXD 4.19 from snap on Fedora 35, or LXD 4.19 on Ubuntu 20.04 using the example inventory. And in any case, this is just an example (and xroadlxdrhss1 reads quite badly).

Reply:
Could you revert the changes to lxd_hosts.txt (or please provide more details of the problem). I was not able to reproduce the issue regarding host names.